nix/lib/getEnvWithDefault.nix

```diff
@@ -0,0 +1,11 @@
+# helper for getting environment variables in a safe way
```

Comment:
I don't think this comment gives much value. The name of the function alone describes what this does well.